PayPal/src/main/java/com/braintreepayments/api/PayPalVaultRequest.java

```diff
@@ -14,6 +14,8 @@ public class PayPalVaultRequest extends PayPalRequest implements Parcelable {
 
     private boolean shouldOfferCredit;
 
+    private String payerEmail;
```

Comment:
We started a discussion on the iOS PR about naming, not sure if you have thoughts but we should align the name across platforms: https://github.com/braintree/braintree_ios/pull/1188#discussion_r1486758270

Reply:
Thanks for calling this out! I've added a do not merge label, will make sure to align with iOS before this goes in.